doc/nrf/gs_programming.rst

```diff
@@ -178,31 +178,78 @@ For more information on building and programming using the command line, see the
+|                   |            |                                                                 | ``nrf9160dk_nrf9160ns``               |
+|                   |            +-----------------------------------------------------------------+---------------------------------------+
+|                   |            | :ref:`nrf9160dk_nrf52840 <zephyr:nrf9160dk_nrf52840>`           | ``nrf9160dk_nrf52840``                |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+
+
+Boards included in sdk-nrf
+==========================
+
+The following boards are defined in the :file:`nrf/boards/arm/` folder.
+
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
 | Hardware platform | PCA number | Board name                                               | Build target                          |
 +===================+============+==========================================================+=======================================+
-| nRF51 DK          | PCA10028   | :ref:`nrf51dk_nrf51422 <zephyr:nrf51dk_nrf51422>`        | ``nrf51dk_nrf51422``                  |
+| nRF Desktop       | PCA20041   | :ref:`nrf52840gmouse_nrf52840 <nrf_desktop>`             | ``nrf52840gmouse_nrf52840``           |
+| Gaming Mouse      |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF52 DK          | PCA10040   | :ref:`nrf52dk_nrf52832 <zephyr:nrf52dk_nrf52832>`        | ``nrf52dk_nrf52832``                  |
+| nRF Desktop       | PCA20045   | :ref:`nrf52810dmouse_nrf52810 <nrf_desktop>`             | ``nrf52810dmouse_nrf52810``           |
+| Mouse             |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF52833 DK       | PCA10100   | :ref:`nrf52833dk_nrf52833 <zephyr:nrf52833dk_nrf52833>`  | ``nrf52833dk_nrf52833``               |
+| nRF Desktop       | PCA20037   | :ref:`nrf52kbd_nrf52832 <nrf_desktop>`                   | ``nrf52kbd_nrf52832``                 |
+| Keyboard          |            |                                                          |                                       |
 +-------------------+------------+----------------------------------------------------------+---------------------------------------+
-| nRF52840 DK       | PCA10056   | :ref:`nrf52840dk_nrf52840 <zephyr:nrf52840dk_nrf52840>`  | ``nrf52840dk_nrf52840``               |
+| nRF Desktop       | PCA20044   | :ref:`nrf52dmouse_nrf52832 <nrf_desktop>`                | ``nrf52dmouse_nrf52832``              |
```

Comment:
Is there a reason for this order? I would list all mouse devices first, then the keyboard.